packages/eslint-plugin/src/rules/no-implied-eval.ts

```diff
@@ -76,14 +76,8 @@ export default createRule({
       }
 
       // eslint-disable-next-line @typescript-eslint/no-unsafe-enum-comparison
```

Comment:
```suggestion
```

It looks like this `eslint-disable` directive is no longer required.

Reply:
addressed :)